test/src/e2e_vm_tests/test_programs/should_pass/language/u64_ops/src/main.sw

```diff
@@ -0,0 +1,54 @@
+script;
+
+use core::ops::*;
+use std::assert::assert;
+
+fn main() -> bool {
+    // 0b0000_1111 = 15
+    // 0b0101_0101 = 85
+    // 0b1010_1010 = 170
+    // 0b1111_0000 = 240
+    // 0b1111_1111 = 255
+    let max = 18446744073709551615; // 0b11111111_11111111_11111111_11111111_11111111_11111111_11111111_11111111
+    let A = 9331882296111890817; // 0b10000001_10000001_10000001_10000001_10000001_10000001_10000001_10000001
+    let B = 9114861777597660798; // 0b01111110_01111110_01111110_01111110_01111110_01111110_01111110_01111110
+
+    assert(~u64::binary_and(0, 0) == 0);
```

Comment:
Looks good! One q: does the compiler not support `&`, `|`, and `^` yet?

Reply:
good question; I will test this now.

Reply:
@adlerjohn 

`^` is not yet supported.
Will open an issue. fixed the others.

Reply:
```
assert(0 ^ 0 == 0);
   |            ^ No method named "xor" found for type "u64".
```